src/app/(protected)/analises/pessoas/consultar/process.tsx

```diff
@@ -83,13 +83,20 @@ const MemoizedCard = memo(Card);
 export function ProcessFinished({ analysis_info, document }: Props) {
   const modal = useModal();
 
+  const [haveProcess, setHaveProcess] = useState(false)
+
   const processList = useMemo(() => {
     try {
       const response = JSON.parse(analysis_info) as ProcessResponse;
 
+      response?.processos_judiciais_administrativos?.total_processos >= 0 ? setHaveProcess(true) : setHaveProcess(false)
```

Comment:
Hmm eu não faria assim, nunca é bom setar um state dentro do useMemo. Eu trabalharia com 3 returns no useMemo (com processos, vazio e nulo).
```suggestion
      response?.processos_judiciais_administrativos?.total_processos >= 0 ?? null
```